webpack/assets/javascripts/react_app/components/HostsIndex/Columns/core.js

```diff
@@ -94,10 +94,14 @@ const coreHostsIndexColumns = [
   {
     columnName: 'last_report',
     title: __('Last report'),
-    wrapper: hostDetails => {
+    wrapper: ({ name, hostDetails }) => {
       const lastReport = hostDetails?.last_report;
-      return (
-        <RelativeDateTime defaultValue={__('Unknown')} date={lastReport} />
+      return lastReport ? (
+        <Link to={`hosts/${name}/config_reports/last`}>
+          <RelativeDateTime date={lastReport} />
+        </Link>
+      ) : (
+        __('Unknown')
```

Comment:
Just thinking out loud here: if `hostDetails` is not undefined (so the host is loaded) and `last_report` is empty then we can conclude there is no last report. In that case I'd prefer it to be explicit, rather than unknown.

Not a blocker.